MAUI/Themes/Keys.md

```diff
@@ -294,4 +294,1497 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the stroke of needle pointer knob.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            SfRadialGaugeNeedlePointerNeedleFillColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the needle pointer fill.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	    <tr>
+        <td>
+            SfRadialGaugeNeedlePointerTailFillColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the tail pointer fill.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	    <tr>
+        <td>
+            SfRadialGaugeRangePointerFillColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the need pointer fill.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	    <tr>
+        <td>
+            SfRadialGaugeRangePointerGradientColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the range point gradient in the radial gauge.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	    <tr>
+        <td>
+            SfRadialGaugeRadialRangeFillColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of radial range fill.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	    <tr>
+        <td>
+            SfRadialGaugeRadialRangeGradientColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the gradient of gauge.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	    <tr>
+        <td>
+            SfRadialGaugeRadialRangeLabelTextColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the radial range label text color.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	    <tr>
+        <td>
+            SfRadialGaugeShapeShadowColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the shape shadow.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	</tr>
+</table>
+
+## SfLinearGauge
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="14">
+            SfLinearGaugeThemeStyles
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfLinearGaugeTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfLinearGauge without merging common theme resource and control style resource dictionaries.
```

Comment:
By merging this key in application resources, you can customize the appearance of SfLinearGauge without merging common theme resources and control style resource dictionaries.